DEVELOP.rst

```diff
@@ -110,25 +110,27 @@ And start Crate::
     ./app/build/install/crate/bin/crate
 
 
-Other common tasks are:
+Other common tasks are::
 
- - Running tests during development::
 
-    ./gradlew --parallel :sql:test -PtestForks=2 itest gtest
+    ./gradlew --parallel -PtestForks=2 :sql:test
```

Comment:
never had a problem with that. Afaik `--parallel` is for the gradle build process and `-PtestForks` is for the actual test execution